src/devtools/client/debugger/src/components/Editor/Breakpoints/Breakpoint.js

```diff
@@ -48,7 +49,7 @@ class Breakpoint extends PureComponent {
   }
 
   onClick = event => {
-    const { cx, breakpoint, removeBreakpointsAtLine } = this.props;
+    const { cx, breakpoint, removeBreakableBreakpointsAtLine } = this.props;
```

Comment:
this feels like it will age poorly. We should probably use `breakPoint` and `logPoint` in the codebase in which case this would be `removeBreakpoint`